crypto/objects/objects.txt

```diff
@@ -933,11 +933,39 @@ nist_hashalgs 1		: SHA256		: sha256
 nist_hashalgs 2		: SHA384		: sha384
 nist_hashalgs 3		: SHA512		: sha512
 nist_hashalgs 4		: SHA224		: sha224
+nist_hashalgs 5		: id-sha512-224		: sha512-224
+nist_hashalgs 6		: id-sha512-256		: sha512-256
```

Comment:
If the SHA3 uses the id- then the SHA2 should also do so:
from PKCS 1 v2.2 one can see:
```
id-sha224    OBJECT IDENTIFIER ::= { 
    joint-iso-itu-t (2) country (16) us (840) organization (1) gov (101) 
    csor (3) nistalgorithm (4) hashalgs (2) 4 
} 
id-sha256    OBJECT IDENTIFIER ::= { 
    joint-iso-itu-t (2) country (16) us (840) organization (1) gov (101) 
    csor (3) nistalgorithm (4) hashalgs (2) 1 
} 
id-sha384    OBJECT IDENTIFIER ::= { 
    joint-iso-itu-t (2) country (16) us (840) organization (1) gov (101) 
    csor (3) nistalgorithm (4) hashalgs (2) 2 
} 
id-sha512    OBJECT IDENTIFIER ::= { 
    joint-iso-itu-t (2) country (16) us (840) organization (1) gov (101) 
    csor (3) nistalgorithm (4) hashalgs (2) 3 
} 
id-sha512-224    OBJECT IDENTIFIER ::= { 
    joint-iso-itu-t (2) country (16) us (840) organization (1) gov (101) 
    csor (3) nistalgorithm (4) hashalgs (2) 5 
} 
id-sha512-256    OBJECT IDENTIFIER ::= { 
    joint-iso-itu-t (2) country (16) us (840) organization (1) gov (101) 
    csor (3) nistalgorithm (4) hashalgs (2) 6 
}
```